pkgs/by-name/ha/harlequin/package.nix

```diff
@@ -65,15 +66,11 @@ python3Packages.buildPythonApplication rec {
     export HOME=$(mktemp -d)
   '';
 
-  nativeCheckInputs =
-    [
-      # FIX: restore on next release
-      # versionCheckHook
-    ]
-    ++ (with python3Packages; [
-      pytest-asyncio
-      pytestCheckHook
-    ]);
+  nativeCheckInputs = with python3Packages; [
+    pytest-asyncio
+    pytestCheckHook
+    versionCheckHook
```

Comment:
please move versionCheckHook to nativeInstallCheckInputs and enable that phase

Reply:
woa, i seem to have been in the wrong, The python builders are special and the version check hook goes in nativeCheckInputs. Sorry, please revert the change :)